src/org_parser/transform.cljc

```diff
@@ -36,13 +36,20 @@
 #_(transform (org-parser.parser/parse "* hello\n** world\n\nasdf"))
 
 
+(defn- property-node
```

Comment:
Without looking into it deeply but this might be a misunderstanding. I think the "prop" parser function was meant for "property nodes" in a property drawer.

It would be better to restore the property function and adding a new function like `todo-keyword` or how it's called in org mode jargon.

Reply:
ah, OK. I totally just guessed at what this was for, and was happy when it worked.

I can restore the property function and write a similar function for capturing TODOs, comments, and priority. Is there any issue you see with the actual implementation, i.e. would this potentially have false positives when it encounters a similar keyword in the `:PROPERTIES:` drawer?

Reply:
Actually, I'm confused why then `property` is used for the `:level` when I think that's based on the number of `*` before the heading?

Reply:
Yeah I just did quick test and here's the output given the changes on this branch:

```clojure
  (core/read-str "
#+keyword: TODO 
* foo bar
:PROPERTIES:
:level: 2
:keyword: TODO
:END:")
  ;; => {:preamble
  ;;     {:section
  ;;      {:ast
  ;;       [[:empty-line]
  ;;        [:keyword-line
  ;;         [:keyword-key "keyword"]
  ;;         [:keyword-value "TODO "]]],
  ;;       :raw ["" "#+keyword: TODO "]}},
  ;;     :headlines
  ;;     [{:headline
  ;;       {:level 1,
  ;;        :title [[:text-normal "foo bar"]],
  ;;        :planning [],
  ;;        :keyword nil,
  ;;        :priority nil,
  ;;        :commented? false,
  ;;        :tags []},
  ;;       :section
  ;;       {:ast
  ;;        [[:drawer-begin-line [:drawer-name "PROPERTIES"]]
  ;;         [:text [:text-normal ":level: 2"]]
  ;;         [:text [:text-normal ":keyword: TODO"]]
  ;;         [:drawer-end-line]],
  ;;        :raw [":PROPERTIES:" ":level: 2" ":keyword: TODO" ":END:"]}}]}

```